Common/xml/Objects/object-form-view.sublime-snippet

```diff
@@ -1,25 +1,25 @@
 <snippet>
-	<content><![CDATA[
+    <content><![CDATA[
 <record model="ir.ui.view" id="${1:name_id}_form">
```

Comment:
the same af the def in python methods... is better to have this just after de CDATA[